packages/common/src/adapters/track.ts

```diff
@@ -151,3 +151,64 @@ export const userTrackMetadataFromSDK = (
 
   return newTrack
 }
+
+export const stemTrackMetadataFromSDK = (
```

Comment:
cc @raymondjacobson I know you have a PR to add the public stems endpoint. I had to write this adapter for our existing usage of the full version. I'm not entirely sure why we need to represent a stem as a full track with mostly null fields (upload validation?). But you'll probably want this to process the response.